packages/manager/modules/pci/src/projects/project/kubernetes/details/audit-logs/audit-logs.routing.js

```diff
@@ -4,11 +4,22 @@ export default /* @ngInject */ ($stateProvider) => {
     views: {
       kubernetesView: 'pciProjectKubernetesDetailsAuditLogs',
     },
+    redirectTo: (transition) =>
+      transition
+        .injector()
+        .getAsync('isFeatureAvailable')
```

Comment:
Use parent feature flipping instead
```suggestion
        .getAsync('isLogToCustomerFeatureAvailable')
```